vt100/Makefile

```diff
@@ -1,5 +1,5 @@
 all: vt100
-CFLAGS=-O3
+CFLAGS?=-O3
```

Comment:
Is this related to -DSHADERDIR?

Reply:
Yes.  This allows `CFLAGS` to be overwritten from the environment.  It is not strictly needed; I can also maintain this change locally as a FreeBSD only patch.

Reply:
Tanks, just checking.